programs/pluto/connections.c

```diff
@@ -214,8 +214,8 @@ delete_sr(struct connection *c, struct spd_route *sr)
  *
  */
 
-void
-delete_connection(struct connection *c, bool relations)
+static void
+__delete_connection(struct connection *c, bool relations, bool force)
```

Comment:
I don't think we use __ to hide private functions elsewhere.
I would call it, "really_delete_connection()", but I don't care that much.